cmd/geth/config.go

```diff
@@ -294,7 +294,105 @@ func makeFullNode(ctx *cli.Context) (*node.Node, ethapi.Backend) {
 		// rather than working correctly until then and freezing at activation until TBC is caught up.
 		// Would rather have node delay full startup to fully sync TBC to required genesis height
 		// than startup without full information and later freeze at hVM Phase 0 activation time.
+
 		for {
+
+			// wait until we have a block header that is above hvm genesis height, then confirm we
+			// have hvm genesis block at the expected height
+			for {
+				select {
+				case <-time.After(1 * time.Second):
+					bh, _, err := vm.TBCFullNode.BlockHeaderBest(ctx.Context)
+					if err != nil {
+						log.Crit(fmt.Sprintf("error getting block header best: %s", err))
```

Comment:
Does that error make sense as written?  `block header best`?

Reply:
@jcvernaleo what were you thinking as an alternative? 🤔

Reply:
I think `best block header` would be more understandable. BlockHeaderBest is the function and makes sense in code, but it reads weirdly like this.

Reply:
ah ok, I can update 👍